executor-service/src/main/java/executor/service/service/Impl/StepExecutionClickCssImpl.java

```diff
@@ -0,0 +1,21 @@
+package executor.service.service.Impl;
+
+import executor.service.model.Step;
+import executor.service.service.StepExecutionClickCss;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class StepExecutionClickCssImpl implements StepExecutionClickCss {
```

Comment:
There'll be different step execution services, they'll all have common interface and must implement 'StepExecution'. So, rename 'StepExecutionClickCss' to 'StepExecution'. And it would be better to remove 'Impl' at the end of the 'StepExecutionClickCssImpl', then.

Reply:
Done